lib/tasks/data_migrations/invoice.rake

```diff
@@ -0,0 +1,32 @@
+namespace :data_migrations do
+  desc 'Copy associated BillingProfile address to Invoice'
+  task populate_billing_fields: :environment do
+    @invoices = Invoice.all
+    migrated_invoice_count = 0
+    failed_invoices = []
+
+    @invoices.each do |invoice|
+      next if invoice.billing_profile.blank?
+      next unless invoice.recipient.nil?
+
+      fields = %w[vat_code street city state postal_code alpha_two_country_code]
```

Comment:
We've already got this [here](https://github.com/internetee/auction_center/pull/399/files#diff-fcf17760e7a27b92c25b76c075e48b86R52). How about to just reuse this method?